relay-kafka/Cargo.toml

```
@@ -0,0 +1,35 @@
+[package]
+name = "relay-kafka"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Kafka related functionality for Relay"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "22.9.0"
```

The version is outdated.

```suggestion
version = "22.10.0"
```

Please also note that there's a self-hosted release today, so ideally we merge before that.